ui-v2/app/controllers/dc/kv/folder.js

```diff
@@ -0,0 +1,2 @@
+import Controller from './index';
+export default Controller.extend();
```

Comment:
Added just for explicitness?

Reply:
Not sure I follow? I'm extending my `index` or my 'listing' controller. Not sure if you can tell ember to use a different controller than the conventional one for a route? Anyway, shout me later